{"variants":[{"paths":["\/documentation\/vigramsdk\/authenticationerror\/tokennotexist(_:)"],"traits":[{"interfaceLanguage":"swift"}]}],"schemaVersion":{"major":0,"minor":3,"patch":0},"hierarchy":{"paths":[["doc:\/\/com.vigram.VigramSDK\/documentation\/VigramSDK","doc:\/\/com.vigram.VigramSDK\/documentation\/VigramSDK\/AuthenticationError"]]},"abstract":[{"type":"text","text":"Error authentication - token is not exist"}],"primaryContentSections":[{"kind":"declarations","declarations":[{"platforms":["iOS"],"languages":["swift"],"tokens":[{"text":"case","kind":"keyword"},{"kind":"text","text":" "},{"text":"tokenNotExist","kind":"identifier"},{"kind":"text","text":"("},{"text":"String","kind":"typeIdentifier","preciseIdentifier":"s:SS"},{"kind":"text","text":")"}]}]}],"sections":[],"kind":"symbol","identifier":{"url":"doc:\/\/com.vigram.VigramSDK\/documentation\/VigramSDK\/AuthenticationError\/tokenNotExist(_:)","interfaceLanguage":"swift"},"metadata":{"symbolKind":"case","role":"symbol","fragments":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"kind":"identifier","text":"tokenNotExist"},{"kind":"text","text":"("},{"preciseIdentifier":"s:SS","kind":"typeIdentifier","text":"String"},{"kind":"text","text":")"}],"modules":[{"name":"VigramSDK"}],"title":"AuthenticationError.tokenNotExist(_:)","roleHeading":"Case","externalID":"s:9VigramSDK19AuthenticationErrorO13tokenNotExistyACSScACmF"},"references":{"doc://com.vigram.VigramSDK/documentation/VigramSDK":{"role":"collection","images":[{"identifier":"logo.png","type":"icon"}],"url":"\/documentation\/vigramsdk","type":"topic","title":"VigramSDK","abstract":[],"kind":"symbol","identifier":"doc:\/\/com.vigram.VigramSDK\/documentation\/VigramSDK"},"doc://com.vigram.VigramSDK/documentation/VigramSDK/AuthenticationError/tokenNotExist(_:)":{"kind":"symbol","fragments":[{"kind":"keyword","text":"case"},{"text":" ","kind":"text"},{"text":"tokenNotExist","kind":"identifier"},{"text":"(","kind":"text"},{"preciseIdentifier":"s:SS","kind":"typeIdentifier","text":"String"},{"text":")","kind":"text"}],"title":"AuthenticationError.tokenNotExist(_:)","abstract":[{"text":"Error authentication - token is not exist","type":"text"}],"role":"symbol","type":"topic","url":"\/documentation\/vigramsdk\/authenticationerror\/tokennotexist(_:)","identifier":"doc:\/\/com.vigram.VigramSDK\/documentation\/VigramSDK\/AuthenticationError\/tokenNotExist(_:)"},"logo.png":{"type":"image","variants":[{"traits":["1x","light"],"url":"\/images\/com.vigram.VigramSDK\/logo.png"}],"identifier":"logo.png","alt":"Vigram logo."},"doc://com.vigram.VigramSDK/documentation/VigramSDK/AuthenticationError":{"role":"symbol","url":"\/documentation\/vigramsdk\/authenticationerror","fragments":[{"text":"enum","kind":"keyword"},{"text":" ","kind":"text"},{"text":"AuthenticationError","kind":"identifier"}],"title":"AuthenticationError","type":"topic","abstract":[{"type":"text","text":"AuthenticationError contains custom failure events when connecting to Authentication process."}],"identifier":"doc:\/\/com.vigram.VigramSDK\/documentation\/VigramSDK\/AuthenticationError","kind":"symbol","navigatorTitle":[{"text":"AuthenticationError","kind":"identifier"}]}}}